### PR TITLE
Fix test running with Electron imported to actions

### PR DESCRIPTION
**Pull request by jakereps**
Also uglify webpacked electron config file as it cuts 66% of the filesize.

Fixed terminal side on our machines, but had to edit to get to work on travis.

### CODE REVIEW

File: app/js/actions/jobs.js [hunk 1]
```diff
@@ -4,9 +4,8 @@ import { makeB64Digest } from '../util/auth';
 // global variable to set and clear intervals for scanning for jobs.
 let jobInterval;
 
-export const newActiveJob = (workflow, job) => ({
+export const newActiveJob = (job) => ({
     type: 'NEW_ACTIVE_JOB',
-    workflow,
```
Comment (ebolyen) [inline app/js/actions/jobs.js]: was `workflow` never used?

Comment (jakereps) [inline app/js/actions/jobs.js]: Yeah, it is unused. I caught it while writing the new tests.

**Comment by ebolyen** [comment]
Looks great @jakereps, would you mind collapsing these commits?

**Comment by jakereps** [comment]
Thanks, @ebolyen. Should be good to go on this one!

**Pull request merged**